RealmSwift/Sync.swift

```diff
@@ -419,6 +419,26 @@ public typealias SyncPermission = RLMSyncPermission
  */
 public typealias SyncAccessLevel = RLMSyncAccessLevel
 
+/// :nodoc:
+public final class _SyncPermissionChange: Object {
```

Comment:
[SwiftLint complains about this name](https://ci.realm.io/job/objc_pr/5771/configuration=Release,target=swiftlint,xcode_version=9.0/consoleFull#-709143373594bc32c4f0ccc7b2ba4d264):
> RealmSwift/Sync.swift:423:14: error: Type Name Violation: Type name should only contain alphanumeric characters: '_SyncPermissionChange' (type_name)